testsuite/MDAnalysisTests/analysis/test_leaflet.py

```diff
@@ -42,3 +44,11 @@ def test_leaflet_finder(self):
         assert_equal(bottom_heads.indices, np.arange(2521,4670,12), err_msg="Found wrong leaflet lipids")
 
 
+    def test_string_vs_atomgroup_proper(self):
+        from MDAnalysis.analysis.leaflet import LeafletFinder
+        lfls_ag = LeafletFinder(self.universe, self.lipid_heads, pbc=True)
+        lfls_string = LeafletFinder(self.universe, self.lipid_head_string, pbc=True)
+        groups_ag = lfls_ag.groups()
+        groups_string = lfls_string.groups()
+        assert_equal(groups_string[0].indices, groups_ag[0].indices)
+        assert_equal(groups_string[1].indices, groups_ag[1].indices)
```

Comment:
Right, another PR for that specific unit test could always be opened, and not just for this string vs. ag test, but for the general behaviour of `LeafletFinder` -- i.e., there is currently only a single unit test for this module in the code base.